server/models/MenuItem.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+/* Schema */
+let MenuItemSchema = mongoose.Schema({
+  name: String,
+  description: String,
+  price: String,
+  options: Object
+});
+
+/* Model */
+const MenuItem = mongoose.model('MenuItem', MenuItemSchema)
+
+/* Query */
+const getMenu = () => {
+  return MenuItem.find({}, function(err, docs){
+    if (!err) {
+      console.log(docs);
+      process.exit()
```

Comment:
What does process.exit() do?  I haven't seen this before and am just curious.

Reply:
It basically just ends the connection. It was hanging so I looked up ways to fix it and tried that. Might do a disconnect instead.

Reply:
You should not be ending your connection on every query. You'll need to reestablish a connection for your queries every time then, which degrades the user experience (bc it takes longer for every query). Your server should reuse an already established connection (aka connect once, and never disconnect).